### PR TITLE
Issue-470 Add drive and volume object in lib-dmtf

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: lib-dmtf/model/storage.go [hunk 1]
```diff
@@ -146,3 +146,183 @@ type NVMeSMARTCriticalWarnings struct {
 	PowerBackupFailed        bool `json:"PowerBackupFailed,omitempty"`
 	SpareCapacityWornOut     bool `json:"SpareCapacityWornOut,omitempty"`
 }
+
+// Drive schema represents a single physical drive for a system
+type Drive struct {
+	OdataContext                  string             `json:"@odata.context,omitempty"`
+	Etag                          string             `json:"@odata.etag,omitempty"`
+	OdataID                       string             `json:"@odata.id"`
```
Comment (Bharath-KKB) [inline lib-dmtf/model/storage.go:154]: use camel case for this, type etc consistenly

File: lib-dmtf/model/storage.go [hunk 2]
```diff
@@ -146,3 +146,183 @@ type NVMeSMARTCriticalWarnings struct {
...
+}
+
+// Current firmware version of the drive
+type Current struct {
+	VersionString string `json:"VersionString,omitempty"`
+}
+
+// Volume contains the details volume properties
+type Volume struct {
+	OdataContext                     string                   `json:"@odata.context"`
```
Comment (Bharath-KKB) [inline lib-dmtf/model/storage.go:233]: use camel case

**Review by Bharath-KKB — COMMENTED** [review]
fix case in attribute naming

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
fixing attribute naming as per comments

File: lib-dmtf/model/storage.go (hunk 2 above)
Comment (irappa-pattar) [reply lib-dmtf/model/storage.go]: changed

**Review by Bharath-KKB — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

File: lib-dmtf/model/storage.go (hunk 1 above)
Comment (irappa-pattar) [reply lib-dmtf/model/storage.go]: updated

**Pull request merged**